source/sass/buyers-guide/views/homepage.scss

```diff
@@ -121,6 +117,26 @@
           }
         }
 
+        .product-links {
+          @include hover-focus-active {
+            text-decoration: none;
+          }
+
+          @media screen and (max-width: $breakpoint) {
+            .product-company {
+              color: $gray-80;
+              font-size: 9px;
+              line-height: 14px;
```

Comment:
We can use the `scaleText` mixin for settting font-size and line-height
```scss
@include scaleText(9px, 14px);
```

Reply:
ah, great

Reply:
Tangential question: if it sets text size as well as lineheight, could we rename this mixin to `setTextSize` rather than suggesting it relies on ``transform: scale`?

Reply:
I've filed https://github.com/mozilla/foundation.mozilla.org/issues/4739 for that discussion/possible work.

Reply:
Yea we can rename it. This is an old mixin that I meant to rename but kept forgetting 😂 (But I was thinking about just fixing the letter casing so it's `scale-text` instead)

I'm not sure if we wanna use`transform: scale` for this because mobile/desktop font types aren't always upsized/downsized in scale. I also prefer explicitly putting down font size and line height for mobile and desktop so they correspond with how the design files and brand guide were documented (in fixed `px` values). This also makes it easy to tell what font style we are using without going through math calculation lol. 

I can work with the designers for that ticket when the time comes.